drip/tests/test_drips.py

```diff
@@ -440,3 +440,23 @@ def test_apply_multiple_rules_with_aggregation(self):
             model_drip.drip.get_queryset())
 
         self.assertEqual(qs.count(), 4)
+
+
+class UrlsTestCase(TestCase):
+    def test_drip_timeline_url(self):
+        timeline_url = reverse('admin:drip_timeline', kwargs={
+            'drip_id': 1,
+            'into_past': 2,
+            'into_future': 3})
+
+        self.assertEqual(timeline_url, '/admin/drip/drip/1/timeline/2/3/')
+
+    def test_view_drip_email_url_url(self):
+        view_drip_email_url = reverse('admin:view_drip_email', kwargs={
+            'drip_id': 1,
+            'into_past': 2,
+            'into_future': 3,
+            'user_id': 4})
+
+        self.assertEqual(view_drip_email_url,
+                         '/admin/drip/drip/1/timeline/2/3/4/')
```

Comment:
```suggestion
        self.assertEqual(
            view_drip_email_url, '/admin/drip/drip/1/timeline/2/3/4/'
        )
```

Reply:
Yeah, I thought about it, but we have this kind of identation algongside the project, it would become sort of inconsistent don't you think?

Reply:
I would implement this suggestion, and then do another PR to improve all the indentations.
WDYT? If you want I can do it

Reply:
Yeah, thanks @brunomichetti !!! I'll open an Issue for this

Reply:
Opened in #29